packages/core/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "$schema": "https://json.schemastore.org/eslintrc",
+    "root": true,
```

Comment:
its not really a root config, there is one at the top of the repo

Reply:
Yeah but it's a root for this specific package. When linting this package we don't want ESLint to search for potential parent configurations.